lib/active_merchant/billing/gateways/worldpay_us.rb

```diff
@@ -169,7 +169,7 @@ def parse(xml)
         "void" => "ns_void",
       }
 
-      def commit(action, post)
+      def commit(action, url, post)
```

Comment:
I feel like it's a slightly more common pattern to pass options straight to commit.  Then you can call the `url` method further downstream.

Reply:
Fair enough, I'll switch this around. Just felt odd to me, since only that one value was needed.